src/utils/download-file.ts

```diff
@@ -6,9 +6,9 @@
  * the LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable unicorn/prefer-node-protocol */
+/** eslint-disable unicorn/prefer-node-protocol */
```

Comment:
You can remove that comment now that we are actually using 'node:fs' for the import